### DISCUSSION TITLE
Export Data warning icon for too many rows in table

### DISCUSSION THREAD

**Discussion by generalledgercpa** [sql-bi/Bravo#523] [Ideas]
In the Export Data screen, there is a list of all tables with rows and size.  To the left of the table name is an icon that has a warning triangle that says, "This table has too many rows.  Only the first million rows can be exported to Excel.  Use CSV format to export all the rows in the table."

The issue is this warning icon appears for tables that have less than one million rows.  For me, it flagged tables with between 118,000 and 900,000 rows.  It did not flag tables with 82,000 rows or less.  Bravo correctly exported my largest table of 900,254 records to an xlsx file, even though the icon warned the table had too many rows.  The takeaway is the icon gives the false impression that the table is too large to export to Excel.

![Export Data Capture](https://user-images.githubusercontent.com/73620826/199745943-ebb91d1a-11b6-4ef1-b77d-334729f14e36.PNG)

**Comment by albertospelta** [comment]
Hi @generalledgercpa, thanks for reporting! It's a bug and the fix will be available in next release (https://github.com/sql-bi/Bravo/issues/525)
